Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM 169942020521.dkr.ecr.eu-west-1.amazonaws.com/base/node:14-alpine-builder
+
+FROM 169942020521.dkr.ecr.eu-west-1.amazonaws.com/base/node:14-alpine-runtime
+
+CMD ["/app/dist/bin/www.js", "--", "3000"]
+
+EXPOSE 3000
```

Comment:
Should this be 3000, shouldn't our port be 18580 based on this?

'create Dockerfile file with instructions below and adjust port number to one application uses'